envoyproxy/envoy
# http/transport tap: Add a new UDP sink

Commit Message:
Add an UDP sink for HTTP/Transport Tap
Additional Description:
See more detailed information in issue https://github.com/envoyproxy/envoy/issues/36945
Risk Level: low
Testing:
Manual test has been done
Also CPR test is done
see issue 36945 for more information

Docs Changes:
Build is passed
Release Notes:
Platform Specific Features: N/A
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]
